src/tests/Microsoft.PowerFx.Core.Tests/TexlTests.cs

```diff
@@ -3119,5 +3161,33 @@ private static void TestSimpleBindingSuccess(string script, DType expectedType,
             Assert.Equal(expectedType, result._binding.ResultType);
             Assert.True(result.IsSuccess);
         }
+
+        private void TestDeferredTypeBindingWarning(string script, SymbolTable symbolTable = null)
+        {
+            var config = new PowerFxConfig
+            {
+                SymbolTable = symbolTable
+            };
+
+            var engine = new Engine(config);
+            var result = engine.Check(script);
+
+            Assert.True(result.IsSuccess);
+            Assert.True(result._binding.ErrorContainer.HasErrors());
+            Assert.True(result._binding.ErrorContainer.GetErrors().All(error => error.MessageKey.Equals(TexlStrings.WarnUnknownType.Key)));
```

Comment:
>result._binding.ErrorContainer.GetErrors().All(error => error.MessageKey.Equals(TexlStrings.WarnUnknownType.Key))); [](http://example.com/codeflow?start=24&length=115)

Can we assert this on result.Errors directly (rather than result._binding.ErrorContainer...)